airbyte-server/src/main/java/io/airbyte/server/handlers/ArchiveHandler.java

```diff
@@ -72,7 +72,7 @@ public ArchiveHandler(final String version, final ConfigRepository configReposit
 
   public File exportData() {
     try {
-      final Path tempFolder = Files.createTempDirectory(ARCHIVE_FILE_NAME);
+      final Path tempFolder = Files.createTempDirectory(Path.of("/tmp"), ARCHIVE_FILE_NAME);
```

Comment:
did this mostly so i could explore these files easily.

Reply:
Are we reverting these back once we are done?

Reply:
i wasn't planning to. it's not bad practice to explicitly put it in `/tmp`